icebram/icebram.cc

```diff
@@ -384,6 +384,10 @@ int main(int argc, char **argv)
 	if (verbose)
 		fprintf(stderr, "Found and replaced %d instances of the memory.\n", max_replace_cnt);
 
+	if (max_replace_cnt == 0) {
+		fprintf(stderr, "No memory instances were replaced.\n");
+		exit(2);
```

Comment:
Thanks for fixing! I was about to propose a similar patch, but you were quicker.

Any good reason for exit(2) rather than exit (1) like on line 381 above?

Reply:
I’m using a different exit code for in (the unlikely) case that somebody wants to differentiate between different error.